package.json

```diff
@@ -29,5 +30,14 @@
     "which": "^1.1.1"
   },
   "homepage": "https://github.com/jspm/npm",
-  "devDependencies": {}
+  "devDependencies": {
+    "istanbul": "0.3.21",
+    "lodash": "3.10.1",
+    "magicpen-media": "1.5.1",
+    "messy": "6.6.2",
+    "mocha": "2.3.3",
+    "unexpected": "9.16.0",
+    "unexpected-messy": "4.8.1",
```

Comment:
Thanks for reminding me @papandreou! I completely forgot about that.

`npm install` will not add them no. But you get warnings like this if you are running npm@2.

```
npm WARN peerDependencies The peer dependency unexpected-messy@^4.8.0 included from unexpected-mitm will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
```